### PR TITLE
RET-3116

### PR DESCRIPTION
**Pull request by GauravChawlaMOJ**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-3116

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: definitions/json/CaseEventToFields.json [hunk 1]
```diff
@@ -3650,6 +3650,26 @@
     "PageLabel": "Final notes",
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseTypeID": "ET_Scotland",
```
Comment (hsjhita) [inline definitions/json/CaseEventToFields.json:3654]: do we need to show these in the event? these should be auto populated via callback

Comment (GauravChawlaMOJ) [reply definitions/json/CaseEventToFields.json]: removed

File: definitions/json/AuthorisationCaseField.json [hunk 2]
```diff
@@ -13739,6 +13739,90 @@
...
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et1VettingCompletedBy",
+    "UserRole": "caseworker-employment-legalrep-solicitor",
+    "CRUD": "CRU"
+  },
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et1VettingCompletedBy",
+    "UserRole": "caseworker-employment-englandwales",
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField.json:13751]: scotland

File: definitions/json/AuthorisationCaseField.json [hunk 3]
```diff
@@ -13739,6 +13739,90 @@
...
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et1VettingCompletedBy",
+    "UserRole": "caseworker-employment-etjudge",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeId": "ET_Scotland",
+    "CaseFieldID": "et1VettingCompletedBy",
+    "UserRole": "caseworker-employment-etjudge-englandwales",
```
Comment (hsjhita) [inline definitions/json/AuthorisationCaseField.json:13775]: scotland

Comment (hsjhita) [reply definitions/json/AuthorisationCaseField.json:13775]: and for the et1DateCompleted below

Comment (hsjhita) [reply definitions/json/AuthorisationCaseField.json:13775]: and judge

File: definitions/json/AuthorisationCaseField.json (hunk 2 above)
Comment (GauravChawlaMOJ) [reply definitions/json/AuthorisationCaseField.json]: thanks, fixed them all

**Pull request merged**